tests/transformers/test_relevant_feature_extractor.py

```diff
@@ -67,25 +68,44 @@ def test_impute_works(self):
         self.assertEqual(list(transformed_X.columns), [])
         self.assertEqual(list(transformed_X.index), list(X.index))
 
+    def test_evaluate_only_added_features_true(self):
+        """
+        The boolean flag `evaluate_only_extracted_features` makes sure that only the time series based features are
+        filtered. This unit tests checks that
+        """
+
+        augmenter = RelevantFeatureAugmenter(feature_extraction_settings=self.extraction_settings,
+                                             evaluate_only_added_features=True,
+                                             column_value="val", column_id="id", column_sort="sort", column_kind="kind")
+
+        y = pd.Series({10: 1, 500: 0})
+        X = pd.DataFrame(index=[10, 500])
+        X["pre_feature"] = 0
+
+        augmenter.set_timeseries_container(self.test_df)
+        augmenter.fit(X, y)
+        transformed_X = augmenter.transform(X.copy())
+
+        self.assertEqual(sum(["pre_feature" == x for x in transformed_X.columns]), 1)
```

Comment:
I want to make sure that is contained exactly one time. 

I want to prevent bugs where the non-time series feature is duplicated.

If you come up with a more elegant way to check that, I would be happy to hear it